coderd/database/migrations/000066_template_version_created_by.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE template_versions ALTER COLUMN created_by SET NOT NULL
```

Comment:
This needs to also set all `NULL` values to a zero uuid.

Reply:
How this can be made?

Reply:
set created by to `'00000000-0000-0000-0000-000000000000'::uuid` when the column is null

Reply:
Where and how should I do that? I'm not understanding 😕

Reply:
before the alter table
`UPDATE template_versions SET created_by = '00000000-0000-0000-0000-000000000000'::uuid WHERE created_by IS NULL`

Reply:
Ahhh I got it. Thanks for the help!

Reply:
also needs a `BEGIN` and `END` to make it a transaction

Reply:
`BEGIN` and `COMMIT` right?

Reply:
ah yea you're right!